torch/optim/chain_lr_scheduler.py

```diff
@@ -0,0 +1,165 @@
+import math
+
+from .lr_scheduler import LambdaLR, StepLR, MultiStepLR, ExponentialLR, CosineAnnealingLR
+
+
+class LambdaChainLR(LambdaLR):
+    """Sets the learning rate of each parameter group to the initial lr
```

Comment:
Now that we've split the ChainLRs into their own modules, it's worth explicitly rewriting the docs to be more clear about what exactly they do (and how they differ from the regular LR classes). Specifically, (1) there should probably be some module level overall description of how chain LRs work, and (2) we can probably drop the closed form equations from these classes (and instead just reference their corresponding non-closed version as a reference.)